package.json

```diff
@@ -68,17 +74,17 @@
     "not ie <= 11",
     "not op_mini all"
   ],
+  "transform": {
+    "\\.js$": "<rootDir>/node_modules/babel-jest"
```

Comment:
What is this for?

Reply:
it was for solving the unexpected token error